### PR TITLE
Update PPO sentiment LLaMA script arguments

### PR DESCRIPTION
**Pull request by meg-tong**

### CODE REVIEW

File: examples/ppo_sentiments_llama.py [hunk 1]
```diff
@@ -101,13 +103,16 @@ def reward_fn(samples: List[str], **kwargs) -> List[float]:
 
     # Take few words off of movies reviews as prompts
     imdb = load_dataset("imdb", split="train+test")
-    prompts = [" ".join(review.split()[:4]) for review in imdb["text"]]
-
+    train_prompts = [" ".join(review.split()[:4]) for review in imdb["text"]]
+    eval_prompts = ["I don't know much about Hungarian underground"] * args.num_eval_prompts
```
Comment (lukasberglund) [inline examples/ppo_sentiments_llama.py:107]: Maybe add a comment about where this text comes from!

**Review by lukasberglund — APPROVED** [review]
Looks good!

**Comment by lukasberglund** [comment]
Out of curiosity, are the prompts now just the beginnings of movie reviews instead of dialogues between llama and the assistant?

**Comment by meg-tong** [comment]
> Out of curiosity, are the prompts now just the beginnings of movie reviews instead of dialogues between llama and the assistant?

The movie reviews is just the example from the original trlx repo! I'll add my dialogue prompts in another script in the `situational-awareness` repo.

**Pull request merged**